specification/purview/resource-manager/Microsoft.Purview/preview/2024-04-01-preview/purview.json

```diff
@@ -1631,6 +1630,64 @@
         }
       }
     },
+    "AccountMergeInfo": {
```

Comment:
Will you be adding any new examples that will show what this looks like? I'm still trying to understand the scenario, and noticed I couldn't find any.

Reply:
Ah, my apologies! I have added this field to the Accounts_Get.json example file. The scenario is also described in a different comment. Hope this helps! :)